content/admin/teamserver/config/integration/SSO.md

```diff
@@ -42,80 +38,78 @@ Use the [Encrypted Editor](admin_tsconfig.html#encrypt) to modify *saml.properti
+* Test the configuration by clicking the **Test** button. If an error occurs, a Contrast provides a debug log for troubleshooting. This test only validates the metadata and Contrast's ability to connect to the IdP.  
+* Once the test is successfull, click **Finish**.
+* Restart Contrast to apply the changes.
 
 <a href="assets/images/SSOConfig.png" rel="lightbox" title="Single Sign-On Configuration"><img class="thumbnail" src="assets/images/SSOConfig.png"/></a>
 
 <a href="assets/images/SSOConfigRestart.png" rel="lightbox" title="Restart TeamServer to Apply Authentication Changes"><img class="thumbnail" src="assets/images/SSOConfigRestart.png"/></a>
 
->**NOTE:** Should you need to edit your configuration later, you may do so inline within the Authentication tab; however, you must re-test and restart TeamServer to apply the changes.
+>**NOTE:** You can edit your configuration later within the Authentication tab; however, you must retest and restart Contrast to apply the changes.
 
+####Public and secret nodes
 
----
-
-## SaaS 
+If [SuperAdmin was disabled](https://docs.contrastsecurity.com/admin_tsconfig.html#access) during installation, you're provided with two sets of metadata: one for the public node and one for the secret node. You need to set up the configuration for both in the UI. 
 
-For SaaS customers, the Contrast System Administrator will configure authentication; however, Organization Administrators may be granted the ability to override these settings. In this case, the override will only allow an organization to switch to using Single Sign-On.
+<a href="assets/images/Configure-SAML-identity-provider.png" rel="lightbox" title="Configure your SAML Identity Provider"><img class="thumbnail" src="assets/images/Configure-SAML-identity-provider.png"/></a>
 
+---
 
-### Notes About Switching to SSO
+## SaaS 
 
-If an Org Admin decides to set up SSO, it is important to note the following:
+For SaaS customers, the Contrast System Administrator configures authentication; however, Organization Administrators may be granted the ability to override these settings. In this case, the override only allows an organization to switch to using SSO.
 
-Contrast does not support user provisioning, therefore all users must have an existing account in TeamServer before authenticating with SSO. Additionally, if users are identified with a user ID rather than an email address, those accounts won’t automatically transfer over to the SSO configuration and will need to be recreated.
+If an Org Admin decides to set up SSO, it's important to note that Contrast doesn't support user provisioning; therefore, all users must have an existing account in Contrast before authenticating with SSO. Also, if users are identified with a user ID rather than an email address, those accounts don’t automatically transfer over to the SSO configuration and must be recreated.
 
-### Setup 
+### Set up 
```

Comment:
If we're going to use as a verb here, should we maybe say "Set up SSO" or "Set up on SaaS"? Also, this header and the one above for the EOP section should match.